Playbooks/playbook-Hybrid-analysis_quick-scan.yml

```diff
@@ -0,0 +1,204 @@
+id: Hybrid-analysis quick-scan
+version: -1
+fromversion: 5.0.0
```

Comment:
Let's verify this, i think it can be 4.5 or even 4.1

Reply:
changed to 4.1